docs/web/authentication/google-sign-in.mdx

```diff
@@ -83,30 +83,41 @@ The Google authentication is automatically enabled when these configurations are
 
 ## Client-Side Implementation
 
-To initiate Google Sign-In, redirect the user to the Google authentication endpoint:
+### Using @modelence/auth-ui (Recommended)
```

Comment:
@omegascorp can we please remove all `auth-ui` references from docs? The app builder relies on the MCP server based on the docs, and otherwise it may use the `auth-ui`, which is not something we want (neither the layout or routing work well when it's so separate). And in general probably worth not mentioning since we are deprecating it anyway.

Reply:
Mentions of auth-ui are moved